implement halloween logo

Tijani-zainab opened the pull request:
Hey randy @ri0nardo 

check this out, please let me know if the width and height were accurate or if I'd need to resize 

![wtt-halloween](https://user-images.githubusercontent.com/64507182/195733271-8b9111d6-8f85-4909-9885-348fe5916fee.png)

ri0nardo commented:
@Tijani-zainab did you see how the logo behaves when you change the screen size to mobile? Even a higher resolution screen?

This looks great. Will look really good when the sidebar is implemented.

jazhen commented on client/src/components/Header/Header.js:
```diff
@@ -21,7 +22,12 @@ const Header = () => {
+            src={WttHalloween}
+            alt="WTT Halloween"
+          />
+          {/* <div className="header-font">Water The Trees</div> */}
```
Can you please remove the commented out code? 

@ri0nardo - I think it would be awesome to have a default, non-themed logo + favicon so we only need to replace the images each time.

Tijani-zainab replied:
Okay @jazhen , will do that now

jazhen commented on client/src/components/Header/Header.scss:
```diff
@@ -15,13 +15,18 @@
-  letter-spacing: -3px;
-  font-size: 1.8rem;
-  word-spacing: 0.6rem;
+// .header-font {
+//   font-family: wtt-thin, Arial, sans-serif, "PT Sans";
+//   color: white;
+//   white-space: nowrap;
+//   letter-spacing: -3px;
+//   font-size: 1.8rem;
+//   word-spacing: 0.6rem;
+// }
```
Same w/ this commented code

Tijani-zainab replied:
> @Tijani-zainab did you see how the logo behaves when you change the screen size to mobile? Even a higher resolution screen?
> 
> This looks great. Will look really good when the sidebar is implemented.

Hi @ri0nardo , the logo seems too large on mobile but I'll like to know if there's any design for it looks on mobile before I go ahead and resize. 

so, I'd get the perfect pixel.

jazhen reviewed (changes requested):
Great work @Tijani-zainab. Could you please address @ri0nardo's and my comments? Everything else looks good to me.

Tijani-zainab commented:
Hi again, hehe
resized the Halloween logo on mobile
let me know what you think  

![wtt-mobile](https://user-images.githubusercontent.com/64507182/195917405-eba42126-ce0a-4c47-afe2-0b493ae8cee7.png)

Tijani-zainab commented:
Hi again, hehe
resized the Halloween logo on mobile
let me know what you think  

![wtt-mobile](https://user-images.githubusercontent.com/64507182/195917405-eba42126-ce0a-4c47-afe2-0b493ae8cee7.png)

ri0nardo commented:
@Tijani-zainab thanks for resizing the logo. I'm going to look at the logo on Figma and see if a centered logo will look better since its taking up alot of screen real estate. Also to me the logo is quite large. I will give you an update once I decide.

ri0nardo commented:
@Tijani-zainab I looked through the site, and didn't like how it was centered. I also found a good sizing system for desktop and mobile. I have a preference in the future to go towards the favicon, but with the horizontal bar that is currently on the site it will look best with the text logo. 

Keep the logo left justified.

I am attaching a jpg of the example of both mobile and desktop.

Desktop the logo size on the svg is 
H: 24 pixels
W: 216 pixels

Mobile the logo size is 
H: 16 pixels
W: 144 pixels

![halloween logo - Desktop Size](https://user-images.githubusercontent.com/98299952/195954012-7779c774-b063-4102-9288-86ff887f48fc.svg)
![halloween logo - Mobile Size](https://user-images.githubusercontent.com/98299952/195954015-7036e3b9-4533-4a05-a243-e00fd8d6865f.svg)

![Mobile Logo](https://user-images.githubusercontent.com/98299952/195954026-06ed72f8-7635-443f-8b70-221d64fc0030.jpg)
<img width="2560" alt="Desktop Logo" src="https://user-images.githubusercontent.com/98299952/195954028-e36a7805-33eb-4a53-a764-7968d96ec0a4.png">

Tijani-zainab commented:
Hey 👋🏾 @ri0nardo 

fixed the logo on mobile 

![wtt-mobile](https://user-images.githubusercontent.com/64507182/195958375-057832fe-8ae8-4498-8c88-22e55d67a10c.png)

ri0nardo commented:
@Tijani-zainab looks great on mobile. The horizontal bar is quite large on mobile, but we may change that in the future. Were you able to implement the desktop svg onto the site?

Great job on it.

Question regarding the logo in general. Were you able to set it so someone could just change the logo to a different theme easily? As in they would just upload the file, and it would appear?

Second question, is there a timer feature where the logo automatically gets replaced from a certain date?

Tijani-zainab commented:
> @Tijani-zainab looks great on mobile. The horizontal bar is quite large on mobile, but we may change that in the future. Were you able to implement the desktop svg onto the site?
> 
> Great job on it.
> 
> Question regarding the logo in general. Were you able to set it so someone could just change the logo to a different theme easily? As in they would just upload the file, and it would appear?
> 
> Second question, is there a timer feature where the logo automatically gets replaced from a certain date?

Hey @ri0nardo 

No, there's no timer for it _changes_ will manually change it, _it's not a big work_
yeah, anyone could easily switch/change the logo

Tijani-zainab commented:
> I would create an archive inside the logos folder for old logos so the only thing that needs to be changed is the old image gets moved to archive/date-header-logo.svg and the new on goes into /assets/images/logos/header-logo.svg Then instead of wttHalloween we just have headerLogo
> 
> import headerLogo from '@/assets/images/logos/header-logo.svg';

Hi, Alrighty

created archive for old logo as you suggested, renamed to `HeaderLogo` and fixed import.

jazhen commented:
The logo is not showing up for me in Firefox.

<img width="553" alt="image" src="https://user-images.githubusercontent.com/6326660/196013669-48253a76-f3e9-4764-8301-43213597af20.png">

I've had this happen to me before and I guess I just found that that Firefox handles Flexbox containers differently that Chrome/Safari. Since the header does not have a set height and the height expands to the objects inside the header, we can't use percentages on the image.

![image](https://user-images.githubusercontent.com/6326660/196013739-d73ad5dc-51ba-4006-ab07-fa4f43b0b4f4.png)

jazhen commented on client/src/components/Header/Header.scss:
```diff
@@ -15,13 +15,9 @@
-  letter-spacing: -3px;
-  font-size: 1.8rem;
-  word-spacing: 0.6rem;
+.header--logo {
+  width: 100%;
+  height: 100%;
```
We can just set the logo size manually according to what @ri0nardo said.

> Desktop the logo size on the svg is
> H: 24 pixels
> W: 216 pixels
>

jazhen commented on client/src/components/Header/Header.scss:
```diff
@@ -77,4 +73,8 @@
   }
+
+  .header--logo {
+    width: 50vw;
```
> Mobile the logo size is
> H: 16 pixels
> W: 144 pixels

Tijani-zainab commented:
> The logo is not showing up for me in Firefox.
> 
> <img alt="image" width="553" src="https://user-images.githubusercontent.com/6326660/196013669-48253a76-f3e9-4764-8301-43213597af20.png">
> 
> I've had this happen to me before and I guess I just found that that Firefox handles Flexbox containers differently that Chrome/Safari. Since the header does not have a set height and the height expands to the objects inside the header, we can't use percentages on the image.
> 
> ![image](https://user-images.githubusercontent.com/6326660/196013739-d73ad5dc-51ba-4006-ab07-fa4f43b0b4f4.png)

Okay, that makes sense

set the sizes by the pixels already, commit the changes.

jazhen commented:
@Tijani-zainab - Thanks! Can you add some screenshots and we can give @ri0nardo the final sign off.

Tijani-zainab commented:
> @Tijani-zainab - Thanks! Can you add some screenshots and we can give @ri0nardo the final sign off.

Okay @jazhen @ri0nardo 

![wtt-desktop](https://user-images.githubusercontent.com/64507182/196039124-85822e2a-4e5b-4b08-88ee-48c9147b10c0.png)

mobile 

![wtt-mobile](https://user-images.githubusercontent.com/64507182/196039384-18b395b4-4881-4560-bd6e-b9a90ada368a.png)

it's looking that small on my small screen, I'm guessing it'll look better on a higher resolution and bigger screen screens.

Tijani-zainab commented:
> @Tijani-zainab - Thanks! Can you add some screenshots and we can give @ri0nardo the final sign off.

Okay @jazhen @ri0nardo 

![wtt-desktop](https://user-images.githubusercontent.com/64507182/196039124-85822e2a-4e5b-4b08-88ee-48c9147b10c0.png)

mobile 

![wtt-mobile](https://user-images.githubusercontent.com/64507182/196039384-18b395b4-4881-4560-bd6e-b9a90ada368a.png)

it's looking that small on my small screen, I'm guessing it'll look better on a higher resolution and bigger screen screens.

ri0nardo commented:
@Tijani-zainab Looks good. The mobile screen size looks off since its not more rectangular, but Im pretty sure its fine. Great Job! If the CSS is good we can close this.

jazhen reviewed (commented):
I don't think the favicon should be part of this PR right?

ri0nardo commented:
The favicon is a seperate issue and pull. #311

jazhen reviewed (changes requested):
Please remove `client/src/assets/images/favicons/halloween-favicon.ico`

Tijani-zainab commented:
> Please remove `client/src/assets/images/favicons/halloween-favicon.ico`
> 
> ![image](https://user-images.githubusercontent.com/6326660/196057884-cc682b05-eebe-46f4-b9a1-a6eb74fd1bde.png)

done ✅

jazhen reviewed (approved):
Thanks for going through all these changes @Tijani-zainab !
